app/openapi.generated.yml

```diff
@@ -10,6 +10,33 @@ tags:
 - name: Health
 - name: User
 paths:
+  /:
+    get:
+      parameters: []
+      responses:
+        '200':
+          content:
+            application/json:
```

Comment:
weird, why did apiflask generate this as application/json? is it a bug/unsupported functionality in apiflask or did we need to configure something?

Reply:
I saw this and was wondering the same thing. I'll take a look at the documentation and see why it did that.

Reply:
Having looked at how the generation works before, you'll need to look in apispec: https://apispec.readthedocs.io/en/latest/ as it is what generates the schema

Alternatively, do you think it would make sense to not include this in the schema? I'm not sure about when you define a route directly, but if you make an `APIBlueprint`, you should be able to pass in `enable_openapi=False` and it shouldn't be a part of Openapi.